app/controllers/provider/admin/backend_apis/stats/base_controller.rb

```diff
@@ -34,6 +34,9 @@ def collection
     current_account.backend_apis.accessible
   end
 
+  # TODO: render deprecated metric.service_id even when attribute is not set
+  # We can consider whether we can change this API as it is not intended for
+  # public use, see 0886ce743ab57c
```

Comment:
I think it should be rendered if exists unless we can fix all users.

I pointed at commit, because from commit you can go to pull request in github. If we move repo though, we will not be able to access pull request anymore and see the commit. Maybe I add pull as well.